csharp/ql/src/utils/model-generator/CaptureSourceModels.qll

```diff
@@ -0,0 +1,32 @@
+private import CaptureSourceModelsSpecific
+private import ModelGeneratorUtils
+
+class FromSourceConfiguration extends TaintTracking::Configuration {
+  FromSourceConfiguration() { this = "FromSourceConfiguration" }
+
+  override predicate isSource(DataFlow::Node source) { sourceNode(source, _) }
+
+  override predicate isSink(DataFlow::Node sink) {
+    exists(TargetApi c |
+      sink instanceof ReturnNodeExt and
+      sink.getEnclosingCallable() = c
```

Comment:
Don't we need a `isRelevantForModels(c)`?

Reply:
No, that is ensured by the characteristic predicate for TargetApi.